backend/main.py

```diff
@@ -1,14 +1,64 @@
 # Simple backend server using flask framework
```

Comment:
Every file containing source code must include copyright and license information. This includes any JS/CSS files that you might be serving out to browsers. (This is to help well-intentioned people avoid accidental copying that doesn't comply with the license.)

Please refer to README.md

Reply:
added